packages/playground/_stories/main/RatingIndicator/RatingIndicator.stories.ts

```diff
@@ -23,42 +24,42 @@ export default {
     argTypes,
 } as Meta<RatingIndicator>;
 
-const Template: UI5StoryArgs<RatingIndicator, StoryArgsSlots> = (args) => html`<div></div>`;
+const Template: UI5StoryArgs<RatingIndicator, StoryArgsSlots> = (args) => html`
+<ui5-rating-indicator
+	value="${ifDefined(args.value)}"
+	max="${ifDefined(args.max)}"
+	?disabled="${ifDefined(args.disabled)}"
+	?readonly="${ifDefined(args.readonly)}"
+	accessible-нame="${ifDefined(args.accessibleName)}"
+></ui5-rating-indicator>`;
 
+export const BasicRatingIndicator = Template.bind({});
+BasicRatingIndicator.args = {
+	value: 3.7
+};
+BasicRatingIndicator.storyName = "Basic Rating Indicator";
 
-export const Template0: StoryFn = () => html`
-<h3>Basic Rating Indicator</h3>
-	<div class="snippet">
-		<ui5-rating-indicator></ui5-rating-indicator>
-		<ui5-rating-indicator value="3"></ui5-rating-indicator>
-		<ui5-rating-indicator value="3.7"></ui5-rating-indicator>
-	</div>
-`;
+export const RatingIndicatorMax = Template.bind({});
+RatingIndicatorMax.args = {
+	value: 5,
+	max: 10
+};
+RatingIndicatorMax.storyName = "Rating Indicator with Max Value";
 
 
-export const Template1: StoryFn = () => html`
-<h3>Rating Indicator With Different Max Value</h3>
-	<div class="snippet">
-		<ui5-rating-indicator max="10" value="5"></ui5-rating-indicator>
-		<ui5-rating-indicator max="3" value="3"></ui5-rating-indicator>
-	</div>
-`;
+export const DisabledRatingIndicator: StoryFn = Template.bind({});
+DisabledRatingIndicator.args = {
+	value: 5,
+	max: 10,
+	disabled: true
+};
+DisabledRatingIndicator.storyName = "Disabled Rating Indicator";
```

Comment:
no need of the story name as the story variable name will result with the same

Reply:
Done